src/testapp/main.cpp

```diff
@@ -22,9 +22,9 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
+
```

Comment:
nitpick: I don't really find it more readable to put a newline after the documentation block, but i don't have any particularly strong objection to it.